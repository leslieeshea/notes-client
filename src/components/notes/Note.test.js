import React from 'react';
import { shallow } from 'enzyme';
import Note from './Note';

describe('Note component', () => {
  it('can render a note', () => {
    const wrapper = shallow(<Note note={{ _id: '123', title: 'hello', body: 'goodbye' }} />);
    
    expect(wrapper).toMatchSnapshot();
  });
});
