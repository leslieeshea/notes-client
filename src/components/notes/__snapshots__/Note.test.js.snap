// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note component can render a note 1`] = `
<p>
  hello
</p>
`;
